notebooks/MAST/HSC/CASJOBS/HSCV3_API/hscv3_api.ipynb

```diff
@@ -0,0 +1,636 @@
+    "    Raises a ValueError exception if there is problem\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    releaselist = (\"v2\", \"v3\")\n",
+    "    if release not in releaselist:\n",
+    "        raise ValueError(f\"Bad value for release (must be one of {', '.join(releaselist)})\")\n",
+    "    if release == \"v2\":\n",
+    "        tablelist = (\"summary\", \"detailed\")\n",
+    "    else:\n",
+    "        tablelist = (\"summary\", \"detailed\", \"propermotions\", \"sourcepositions\")\n",
+    "    if table not in tablelist:\n",
+    "        raise ValueError(f\"Bad value for table (for {release} must be one of {', '.join(tablelist)})\")\n",
+    "    if table == \"summary\":\n",
+    "        magtypelist = (\"magaper2\", \"magauto\")\n",
+    "        if magtype not in magtypelist:\n",
+    "            raise ValueError(f\"Bad value for magtype (must be one of {', '.join(magtypelist)})\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get metadata on available columns <a name=\"metadata\"></a>\n",
+    "\n",
+    "The `metadata` query returns information on the columns in the table.  It works for any of the tables in the API (`summary`, `detailed`, `propermotions`, `sourcepositions`).\n",
+    "\n",
+    "Note that the summary table has a huge number of columns!  Each of the 133 filter/detector combinations has 3 columns with the magnitude, median absolute deviation (MAD, a robust measure of the scatter among the measurements), and the number of independent measurements in the filter.  The filter name includes a prefix for the detector (`A`=ACS/WFC, `W3`=WFC3/UVIS or WFC3/IR, `W2`=WFPC2) followed by the standard name of the filter.  So for instance all three instruments have an F814W filter, so there are columns for `A_F814W`, `W3_F814W`, and `W2_F814W`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meta = hscmetadata(\"summary\")\n",
+    "print(len(meta), \"columns in summary\")\n",
+    "filterlist = meta['name'][19::3].tolist()\n",
+    "print(len(filterlist), \"filters\")\n",
+    "pprint(filterlist, compact=True)\n",
+    "meta[:19]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Find variable objects in the dwarf irregular galaxy IC 1613 <a name=\"ic1613\"></a>\n",
+    "\n",
+    "This is based on [HSC Use Case #3](https://archive.stsci.edu/hst/hsc/help/use_case_3_v2.html), which shows an example of selecting objects from the HSC in portal.  This is simple to do using the HSC API."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Use `astropy` name resolver to get position of IC 1613 <a name=\"resolver\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "target = 'IC 1613'\n",
+    "coord = SkyCoord.from_name(target)\n",
```

Comment:
Maybe `ic1613_coord`?  In many workflows there are several different coordinate objects wandering around a notebook...

Reply:
Tagging @rlwastro 
`coord` is used once here. If the notebook had many different `coord`, I agree, but I don't think it's worth refactoring for one use. Also, it goes into this [comment](https://github.com/spacetelescope/notebooks/pull/203#discussion_r937023804), to use `coord.dec.deg`, which then really becomes `ic1613_coord.dec.deg`, which is verbose compared to `ra` and `dec`, which are used twice, and never reassigned.